pkg/metrics/metrics.go

```diff
@@ -80,6 +87,20 @@ var HcoMetrics = func() hcoMetrics {
 				)
 			},
 		},
+		HCOMetricSystemHealthStatus: {
+			fqName:          "kubevirt_hco_system_health_status",
+			help:            "Indicates whether the system health status is healthy (0), warning (1), or error (2)",
```

Comment:
Is "HCO system health status" same as "system health status"? If not, should we add that bit in the help?

Reply:
The metric aggregates the health of the system - HCO and its secondary resources. We use `hco_` prefix for all HCO metrics, in order to identify and group metrics that are generated by it. 
But sure, it is possible to add more information to the `help`. @sradco WDYT about this description?

Reply:
@assafad Please add this sentence "The metric aggregates the health of the system - HCO and its secondary resources. " to the help message.

Reply:
updated